example/aws-example/main.tf

```diff
@@ -0,0 +1,35 @@
+provider "aws" {
+  region = "${var.region}"
```

Comment:
I think it would be a good idea to pin the provider version even in the examples.